Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTarget.cs

```diff
@@ -0,0 +1,54 @@
+﻿using pdxpartyparrot.Core.Actors;
+
+using UnityEngine;
+
+// TODO: Lots. Basic class implementation so I can reference this in the manager
+namespace pdxpartyparrot.ssjAug2018.DeliveryTargets
+{
+    [RequireComponent(typeof(Collider))]
+    [RequireComponent(typeof(AudioSource))]
+    public sealed class DeliveryTarget : NetworkActor {
```

Comment:
these don't need to be NetworkActors (I was wrong about that in my comment in Trello). they shouldn't be actors at all. they'll be more like Spawnpoints, except that they'll have a model associated with them.